examples/basic_query/example_client.py

```diff
@@ -40,7 +40,7 @@ def signal_handler(signal, frame):
 
 if __name__ == '__main__':
     signal.signal(signal.SIGINT, signal_handler)
-    clipper_conn = ClipperConnection(DockerContainerManager())
+    clipper_conn = ClipperConnection(DockerContainerManager(use_centralized_log=False))
```

Comment:
Isn't it turned off by default?

Reply:
I wanted to expose this option to users by adding it to the example code. If you think it is better removing it, I will do that! Let me know